src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.hpp

```diff
@@ -23,6 +23,7 @@
 #include "physicsSolvers/surfaceGeneration/SurfaceGenerator.hpp"
 #include "physicsSolvers/multiphysics/SinglePhasePoromechanics.hpp"
 #include "physicsSolvers/fluidFlow/SinglePhaseBase.hpp"
+#include "common/LogLevelsInfo.hpp"
```

Comment:
I thing the `LogLevelInfo.hpp` include should either go in all `.cpp`s, or in `SolverBase.hpp`. I would prefer that in the `.cpp`s for allowing targeted `LogLevelInfo.hpp` if that get sense later.